test/github-issues/9152/issue-9152.ts

```diff
@@ -0,0 +1,39 @@
+import "reflect-metadata"
+import { expect } from "chai"
+import { Test, ValueUnion } from "./entity/Test"
+import { DataSource } from "../../../src/data-source/DataSource"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { LessThan } from "../../../src"
+
+describe("github issues > #9152 Can't use LessThan for Union field", () => {
+    let connections: DataSource[]
+    before(
+        async () =>
+            (connections = await createTestingConnections({
+                entities: [Test],
+            })),
+    )
+    beforeEach(() => reloadTestingDatabases(connections))
+    after(() => closeTestingConnections(connections))
+
+    it("should not raise TypeScript error when LessThan with Union is passed to FindOptionsWhere", () =>
```

Comment:
In reality, it's enough to only see that the typecheck succeeds, and there is no need to add tests to be executed.